plugins/buildClusters/get.js

```diff
@@ -23,20 +23,24 @@ module.exports = () => ({
             }
         },
         handler: (request, reply) => {
+            const name = request.params.name;
             const factory = request.server.app.buildClusterFactory;
             const config = {
                 params: {
-                    name: request.params.name
+                    name
                 }
             };
 
             return factory.list(config)
                 .then((model) => {
-                    if (!model) {
-                        return reply(boom.notFound('Build cluster does not exist'));
+                    if (!Array.isArray(model)) {
```

Comment:
No need for this check, we have it in model already.
https://github.com/screwdriver-cd/models/blob/master/lib/baseFactory.js#L154